go/libraries/doltcore/schema/column.go

```diff
@@ -15,22 +15,39 @@
 package schema
 
 import (
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/dolttypes"
 	"math"
 	"strings"
 
 	"github.com/liquidata-inc/dolt/go/store/types"
 )
 
-// KindToLwrStr maps a noms kind to the kinds lowercased name
-var KindToLwrStr = make(map[types.NomsKind]string)
+// FullKind is the combination of both a DoltKind and NomsKind, represents the absolute kind
+type FullKind struct {
```

Comment:
I don't really like this approach. A DoltKind should only ever be backed by one NomsKind.  Why would you need to know anything other than the DoltKind if it can be mapped?